core/lib/mno_enterprise/platform_adapters/nex_adapter.rb

```diff
@@ -33,15 +33,38 @@ class << self
         ]
 
         # @see MnoEnterprise::PlatformAdapters::Adapter#restart
-        def restart
+        def restart(timestamp = nil)
           # TODO: implement using nex_client
           # For now this work with one node
 
           # Restart myself
           # nex_app.restart
           # puts "success" unless nex_app.errors.any?
 
-          FileUtils.touch('tmp/restart.txt')
+          cmd = "touch tmp/restart.txt && " \
+            "timestamp=0 && " \
+            "while [ $timestamp -ne #{timestamp} ]; " \
+            "do aux=$(echo `curl -s -X GET http://localhost/mnoe/config.json`) && " \
+            "timestamp=${aux:-0}; done"
+          c = NexClient::ExecCmd.new(
+            name: "check restart timestamp",
+            script: cmd,
+            local: true,
+            parallel: true
```

Comment:
Does that mean all containers restart at once?
Probably not the best as that might cause some downtime while puma restart?
If you put `parallel: false` does that might the job execute sequentially and will wait for all containers to have restarted before exiting? That'd probably be a better way.

Reply:
Yep, let's do it sequentially